client/src/App.tsx

```diff
@@ -311,6 +318,32 @@ function App() {
 		}
 	}, [network, isLoggedIn]);
 
+	//FETCH USER DATA
+	useEffect(() => {
+		if(accountId && isLoggedIn){
+			/* console.log(`/chatroom-users/get/${accountId}`); */
+			axios.get(`/chatroom-users/get/${accountId}`)
+				.then((res: any) => {
+					/* console.log(res.data); */
+					setUserProfile((profile) => ({ ...profile, name: res.data.screenName, avatar: res.data.avatar }));
+					socket.emit('event', {
+						key: 'avatar',
+						value: res.data.avatar
+					});
+					socket.emit('event', {
+						key: 'username',
+						value: res.data.screenName
+					});
+				})
+				.catch((err: any) => {
```

Comment:
I dont think it should return an error, errors are reserved for bad requests like server-side issues or invalid auth. Should return data but maybe null. This way we can handle actual errors like the server failing for some reason instead of trying to add a user in case of an actual error